### ISSUE TITLE
bootstrapBCA confidence interval limits are switched

### ISSUE THREAD

**Issue by jsermeno** [bos/statistics#108]
Hi,

On master if you call `Statistics.Types.confidenceInterval` on the result of a call to `bootstrapBCA` the lower bound will be the higher bound and vice versa. For example you might receive a tuple like this: `(0.5, -0.5)`.

It looks like it's because of the line `z1 = getNSigma confidenceLevel`. `getNSigma` negates the code that originally used to be there: `z1 = quantile standard ((1 - confidenceLevel) / 2)`.

```
-- | Express confidence level in sigmas
getNSigma :: CL Double -> Double
getNSigma (CL p) = negate $ quantile standard (p / 2)
```

where `p` is equivalent to what used to be `(1 - confidenceLevel)`.

The confidence interval limits are still the same, they're just switched. Since I'm not too familiar with the algorithm I'm not sure what the proper fix is. Briefly skimming the algorithm it looks like the z-value is not related to sigmas and shouldn't be negated. Should we replace `z1 = getNSigma confidenceLevel` with `z1 = quantile standard ((getPValue confidenceLevel) / 2)`?

**Comment by jsermeno** [comment]
Somewhat aside, would `getPValue` be more clear named `getAlphaValue`? Or is it that the `CL` represents both alpha-values and p-values?

**Comment by Shimuuar** [comment]
Thank you. I'll look into it

**Comment by Shimuuar** [comment]
Fixed by e5baa10009c234c974c47ee2d3c015a470b66030
